src/components/calcite-pick-list-item/calcite-pick-list-item.scss

```diff
@@ -20,23 +25,48 @@
   }
 }
 
-:host(.highlight[selected]) {
+.highlight[selected] {
   border-left-color: blue;
   &:host(.calcite--rtl) {
     border-left-color: none;
     border-right-color: blue;
   }
 }
 
+.icon {
+  color: var(--calcite-app-foreground-link);
+  flex: 0 0;
+  line-height: 0;
+  width: var(--calcite-app-icon-size);
+  margin: 0 var(--calcite-app-side-spacing-half);
+}
+
 .label {
-  flex-grow: 1;
-  margin: 0 12px;
-  min-width: 60px;
+  display: flex;
+  flex-flow: column nowrap;
+  flex: 1 0;
```

Comment:
when using the shorthand `flex` property its a good idea to be explicit with all 3 values because the browsers are inconsistent with their defaults, namely Safari.

Reply:
done